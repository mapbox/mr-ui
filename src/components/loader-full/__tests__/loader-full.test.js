import React from 'react';
import { shallow } from 'enzyme';
import { testCases } from './loader-full-test-cases';

describe('LoaderFull', () => {
  let testCase;
  let wrapper;

  describe(testCases.loaderBasic.description, () => {
    beforeEach(() => {
      testCase = testCases.loaderBasic;
      wrapper = shallow(
        React.createElement(testCase.component, testCase.props)
      );
    });

    test('renders', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
