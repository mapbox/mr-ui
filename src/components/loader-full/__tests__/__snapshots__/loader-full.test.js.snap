// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderFull basic renders 1`] = `
<div
  className="fixed top left right bottom flex flex--center-main flex--center-cross bg-darken50"
  data-test="loader"
  style={
    Object {
      "zIndex": 9999,
    }
  }
>
  <div
    className="loading loading--dark"
  />
</div>
`;
